CHANGES/1156.bugfix

```diff
@@ -0,0 +1 @@
+Relaxed the value's requirement for a media type of config objects in Docker V2 manifests.
```

Comment:
Added ``application/octet-stream`` as an accepted media_type for docker config objects